packages/components/organisms/f-checkout/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-checkout",
   "description": "Fozzie Checkout – Fozzie Checkout Component",
-  "version": "0.141.0",
+  "version": "0.141.1",
```

Comment:
Was there any reason this was a patch bump and the others were minor version bumps?

Reply:
I had to fix some visual regression tests in this one 👍🏼

Reply:
I'd say you need to bump the version consistently as this project contained the same changes and more compared to the other package yet they had a minor version bump with fewer changes?